codespell_lib/data/dictionary.txt

```diff
@@ -4701,6 +4711,7 @@ constructer->constructor
 constructers->constructors
 constructes->constructs
 constructred->constructed
+constructt->construct
```

Comment:
Could add the s, ed, or, ors and ing variants of this typo too.

Reply:
900c7b3